Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -222,6 +223,12 @@ MainWindow::MainWindow(std::unique_ptr<BootParameters> boot_parameters,
 
   InitControllers();
 
+#ifdef USE_RETRO_ACHIEVEMENTS
+  // This has to be done before CreateComponents() so it's initialized.
+  Achievements::Init();
+  Achievements::Login();
```

Comment:
Shouldn't there be a check to see if it actually succeeded and log whether or not it failed (or display a dialog, whichever is better)?

Reply:
There's currently no dialogs or visible interface associated with this for the sake of keeping the PR small and reviewable. The check you ask for will come later.